<?xml version="1.0"?>
<launch>
  <!-- Resolution in meter for the map when starting with an empty map.
        Otherwise the loaded file's resolution is used. -->
  <arg name="uav_id"          default="0"/>
  <arg name="uav_name"        default="uav$(arg uav_id)"/>
  <arg name="uav_start_x"     default="0.0"/>
  <arg name="uav_start_y"     default="0.0"/>
  <arg name="uav_takeoff_z"   default="2.0"/>

  <node pkg="mipp_packages" type="uav_informative_explorer_node" name="uav_informative_explorer_node" output="screen">
    <!-- Parameters -->
    <param name="uav_id"            value="$(arg uav_id)"/>
    <param name="uav_start_x"       value="$(arg uav_start_x)"/>
    <param name="uav_start_y"       value="$(arg uav_start_y)"/>
    <param name="uav_takeoff_z"     value="$(arg uav_takeoff_z)"/>
    <param name="uav_world_frame"   value="world"/>
    <param name="uav_local_frame"   value="odom_$(arg uav_name)"/>
    <param name="uav_body_frame"    value="base_link_$(arg uav_name)"/>

    <!-- Remap topics -->
    <!-- Published topics -->
    <!-- Subscribed topics -->
    <remap from="uav_server/ground_truth_uav" to="/gazebo/ground_truth_$(arg uav_name)"/>
  </node>
</launch>

